Yank/sampling.py

```diff
@@ -90,14 +90,12 @@ def __init__(self, store_filename, **kwargs):
 
         """
         super(ModifiedHamiltonianExchange, self).__init__(store_filename, **kwargs)
-        self._reference_context = None
-        self.reference_state = None
-        self.reference_LJ_state = None
-        self._reference_LJ_context = None
-        self.reference_LJ_expanded_state = None
-        self._reference_LJ_expanded_context = None
-
-    def create(self, base_state, alchemical_states, positions, displacement_sigma=None, mc_atoms=None, options=None, metadata=None, reference_state=None, reference_LJ_state=None, reference_LJ_expanded_state=None):
+        self._fully_interacting_expanded_context = None
+        self.fully_interacting_expanded_state = None
+        self._noninteracting_expanded_context = None
+        self.noninteracting_expanded_state = None
+
+    def create(self, base_state, alchemical_states, positions, displacement_sigma=None, mc_atoms=None, options=None, metadata=None, fully_interacting_expanded__state=None, noninteracting_expanded_state=None):
```

Comment:
Here you have the typo in `fully_interacting_expanded__state` (two underscores) that is causing the tests to fail.